--- STDOUT ---
INFO 06-24 23:41:59 [__init__.py:244] Automatically detected platform cuda.
When dataset path is not set, it will default to random dataset
Namespace(backend='vllm', dataset_name='random', dataset=None, dataset_path=None, input_len=256, output_len=256, n=1, num_prompts=100, hf_max_batch_size=None, output_json=None, async_engine=False, disable_frontend_multiprocessing=False, disable_detokenize=False, lora_path=None, prefix_len=None, random_range_ratio=None, hf_subset=None, hf_split=None, model='meta-llama/Llama-3.3-70B-Instruct', task='auto', tokenizer='meta-llama/Llama-3.3-70B-Instruct', tokenizer_mode='auto', trust_remote_code=False, dtype='auto', seed=0, hf_config_path=None, allowed_local_media_path='', revision=None, code_revision=None, rope_scaling={}, rope_theta=None, tokenizer_revision=None, max_model_len=512, quantization=None, enforce_eager=False, max_seq_len_to_capture=8192, max_logprobs=20, disable_sliding_window=False, disable_cascade_attn=False, skip_tokenizer_init=False, enable_prompt_embeds=False, served_model_name=None, disable_async_output_proc=False, config_format='auto', hf_token=None, hf_overrides={}, override_neuron_config={}, override_pooler_config=None, logits_processor_pattern=None, generation_config='auto', override_generation_config={}, enable_sleep_mode=False, model_impl='auto', load_format='auto', download_dir=None, model_loader_extra_config={}, ignore_patterns=None, use_tqdm_on_load=True, qlora_adapter_name_or_path=None, pt_load_map_location='cpu', guided_decoding_backend='auto', guided_decoding_disable_fallback=False, guided_decoding_disable_any_whitespace=False, guided_decoding_disable_additional_properties=False, enable_reasoning=None, reasoning_parser='', distributed_executor_backend=None, pipeline_parallel_size=1, tensor_parallel_size=8, data_parallel_size=1, data_parallel_size_local=None, data_parallel_address=None, data_parallel_rpc_port=None, data_parallel_backend='mp', enable_expert_parallel=False, max_parallel_loading_workers=None, ray_workers_use_nsight=False, disable_custom_all_reduce=False, worker_cls='auto', worker_extension_cls='', enable_multimodal_encoder_data_parallel=False, block_size=None, gpu_memory_utilization=0.9, swap_space=4, kv_cache_dtype='auto', num_gpu_blocks_override=None, enable_prefix_caching=None, prefix_caching_hash_algo='builtin', cpu_offload_gb=0, calculate_kv_scales=False, tokenizer_pool_size=0, tokenizer_pool_type='ray', tokenizer_pool_extra_config={}, limit_mm_per_prompt={}, mm_processor_kwargs=None, disable_mm_preprocessor_cache=False, enable_lora=None, enable_lora_bias=False, max_loras=1, max_lora_rank=16, lora_extra_vocab_size=256, lora_dtype='auto', long_lora_scaling_factors=None, max_cpu_loras=None, fully_sharded_loras=False, enable_prompt_adapter=None, max_prompt_adapters=1, max_prompt_adapter_token=0, device='auto', speculative_config=None, show_hidden_metrics_for_version=None, otlp_traces_endpoint=None, collect_detailed_traces=None, max_num_batched_tokens=None, max_num_seqs=None, max_num_partial_prefills=1, max_long_partial_prefills=1, cuda_graph_sizes=[512], long_prefill_token_threshold=0, num_lookahead_slots=0, scheduler_delay_factor=0.0, preemption_mode=None, num_scheduler_steps=1, multi_step_stream_outputs=True, scheduling_policy='fcfs', enable_chunked_prefill=None, disable_chunked_mm_input=False, scheduler_cls='vllm.core.scheduler.Scheduler', disable_hybrid_kv_cache_manager=False, kv_transfer_config=None, kv_events_config=None, compilation_config={"level":0,"debug_dump_path":"","cache_dir":"","backend":"","custom_ops":[],"splitting_ops":[],"use_inductor":true,"compile_sizes":null,"inductor_compile_config":{"enable_auto_functionalized_v2":false},"inductor_passes":{},"use_cudagraph":true,"cudagraph_num_of_warmups":0,"cudagraph_capture_sizes":null,"cudagraph_copy_inputs":false,"full_cuda_graph":false,"max_capture_size":null,"local_cache_dir":null}, additional_config={}, use_v2_block_manager=True, disable_log_stats=False, disable_log_requests=False)
INFO 06-24 23:42:08 [config.py:823] This model supports multiple tasks: {'score', 'generate', 'embed', 'classify', 'reward'}. Defaulting to 'generate'.
INFO 06-24 23:42:08 [config.py:1946] Defaulting to use mp for distributed inference
INFO 06-24 23:42:08 [config.py:2195] Chunked prefill is enabled with max_num_batched_tokens=16384.
INFO 06-24 23:42:09 [core.py:455] Waiting for init message from front-end.
INFO 06-24 23:42:09 [core.py:70] Initializing a V1 LLM engine (v0.9.1) with config: model='meta-llama/Llama-3.3-70B-Instruct', speculative_config=None, tokenizer='meta-llama/Llama-3.3-70B-Instruct', skip_tokenizer_init=False, tokenizer_mode=auto, revision=None, override_neuron_config={}, tokenizer_revision=None, trust_remote_code=False, dtype=torch.bfloat16, max_seq_len=512, download_dir=None, load_format=LoadFormat.AUTO, tensor_parallel_size=8, pipeline_parallel_size=1, disable_custom_all_reduce=False, quantization=None, enforce_eager=False, kv_cache_dtype=auto,  device_config=cuda, decoding_config=DecodingConfig(backend='auto', disable_fallback=False, disable_any_whitespace=False, disable_additional_properties=False, reasoning_backend=''), observability_config=ObservabilityConfig(show_hidden_metrics_for_version=None, otlp_traces_endpoint=None, collect_detailed_traces=None), seed=0, served_model_name=meta-llama/Llama-3.3-70B-Instruct, num_scheduler_steps=1, multi_step_stream_outputs=True, enable_prefix_caching=True, chunked_prefill_enabled=True, use_async_output_proc=True, pooler_config=None, compilation_config={"level":3,"debug_dump_path":"","cache_dir":"","backend":"","custom_ops":["none"],"splitting_ops":["vllm.unified_attention","vllm.unified_attention_with_output"],"use_inductor":true,"compile_sizes":[],"inductor_compile_config":{"enable_auto_functionalized_v2":false},"inductor_passes":{},"use_cudagraph":true,"cudagraph_num_of_warmups":1,"cudagraph_capture_sizes":[512,504,496,488,480,472,464,456,448,440,432,424,416,408,400,392,384,376,368,360,352,344,336,328,320,312,304,296,288,280,272,264,256,248,240,232,224,216,208,200,192,184,176,168,160,152,144,136,128,120,112,104,96,88,80,72,64,56,48,40,32,24,16,8,4,2,1],"cudagraph_copy_inputs":false,"full_cuda_graph":false,"max_capture_size":512,"local_cache_dir":null}
WARNING 06-24 23:42:09 [multiproc_worker_utils.py:307] Reducing Torch parallelism from 112 threads to 1 to avoid unnecessary CPU contention. Set OMP_NUM_THREADS in the external environment to tune this value as needed.
INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0, 1, 2, 3, 4, 5, 6, 7], buffer_handle=(8, 16777216, 10, 'psm_3002db2a'), local_subscribe_addr='ipc:///tmp/a6894e2c-e904-4cde-89c4-e35c8043c4d6', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x7102117416a0>
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_407a2aab'), local_subscribe_addr='ipc:///tmp/870c6384-2872-439b-9a8b-e965cfb735d7', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x71008ddf6f30>
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_0a3e0c23'), local_subscribe_addr='ipc:///tmp/0d70ab36-e14d-4750-bc5d-511ac797b915', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x71008ddf6990>
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_c5bb247d'), local_subscribe_addr='ipc:///tmp/236923cb-cfa0-418b-83ef-ea0d45c6c4ff', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x71008ddf6090>
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x7102117415b0>
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_edb3078b'), local_subscribe_addr='ipc:///tmp/2a11113b-915b-47c2-98d0-2445892e64e6', remote_subscribe_addr=None, remote_addr_ipv6=False)
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_632827fd'), local_subscribe_addr='ipc:///tmp/886c63eb-290d-4597-a031-686fbf37601e', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x71008ddf6390>
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_5f817afd'), local_subscribe_addr='ipc:///tmp/cb5b54f9-9c57-4ffe-85fc-526088426701', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x7102182316d0>
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_0650fbac'), local_subscribe_addr='ipc:///tmp/257978bd-5d84-49af-83ea-9fe47066c488', remote_subscribe_addr=None, remote_addr_ipv6=False)
WARNING 06-24 23:42:09 [utils.py:2737] Methods determine_num_available_blocks,device_config,get_cache_block_size_bytes,initialize_cache not implemented in <vllm.v1.worker.gpu_worker.Worker object at 0x71008ddf6330>
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:09 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[0], buffer_handle=(1, 10485760, 10, 'psm_acdaa63f'), local_subscribe_addr='ipc:///tmp/6efd02ba-9d2f-4814-8745-e0c27eca22ef', remote_subscribe_addr=None, remote_addr_ipv6=False)
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:20 [utils.py:1126] Found nccl from library libnccl.so.2
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:20 [pynccl.py:70] vLLM is using nccl==2.26.2
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:52 [custom_all_reduce_utils.py:246] reading GPU P2P access cache from /home/pirillo_google_com/.cache/vllm/gpu_p2p_access_cache_for_0,1,2,3,4,5,6,7.json
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:52 [shm_broadcast.py:289] vLLM message queue communication handle: Handle(local_reader_ranks=[1, 2, 3, 4, 5, 6, 7], buffer_handle=(7, 4194304, 6, 'psm_0c9f94ae'), local_subscribe_addr='ipc:///tmp/3cabf8ce-8792-41b2-910f-f8b3edf5fa33', remote_subscribe_addr=None, remote_addr_ipv6=False)
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 1 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 1, EP rank 1
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 0 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 0, EP rank 0
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 7 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 7, EP rank 7
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 6 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 6, EP rank 6
[1;36m(VllmWorker rank=6 pid=16508)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=1 pid=16503)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=0 pid=16502)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=7 pid=16509)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 3 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 3, EP rank 3
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 5 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 5, EP rank 5
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 4 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 4, EP rank 4
[1;36m(VllmWorker rank=3 pid=16505)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=5 pid=16507)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=4 pid=16506)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:52 [parallel_state.py:1065] rank 2 in world size 8 is assigned as DP rank 0, PP rank 0, TP rank 2, EP rank 2
[1;36m(VllmWorker rank=2 pid=16504)[0;0m WARNING 06-24 23:42:52 [topk_topp_sampler.py:59] FlashInfer is not available. Falling back to the PyTorch-native implementation of top-p & top-k sampling. For the best performance, please install FlashInfer.
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1595] Starting to load model meta-llama/Llama-3.3-70B-Instruct...
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:52 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:53 [gpu_model_runner.py:1600] Loading model from scratch...
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:53 [cuda.py:246] FlashInfer failed to import for V1 engine on Blackwell (SM 10.0) GPUs; it is recommended to install FlashInfer for better performance.
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:53 [cuda.py:252] Using Flash Attention backend on V1 engine.
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:42:53 [weight_utils.py:292] Using model weights format ['*.safetensors']
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:43:00 [default_loader.py:272] Loading weights took 6.55 seconds
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:43:00 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 7.361023 seconds
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:43:00 [default_loader.py:272] Loading weights took 6.91 seconds
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:43:00 [default_loader.py:272] Loading weights took 6.88 seconds
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:43:01 [default_loader.py:272] Loading weights took 6.80 seconds
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:43:01 [default_loader.py:272] Loading weights took 7.01 seconds
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:43:01 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 7.903110 seconds
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:43:01 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 7.813449 seconds
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:43:01 [default_loader.py:272] Loading weights took 6.96 seconds
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:43:01 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 8.185153 seconds
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:43:01 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 8.264799 seconds
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:43:01 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 8.387285 seconds
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:43:01 [default_loader.py:272] Loading weights took 7.40 seconds
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:43:02 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 8.883854 seconds
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:43:02 [default_loader.py:272] Loading weights took 8.61 seconds
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:43:02 [gpu_model_runner.py:1624] Model loading took 16.4607 GiB and 9.762736 seconds
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_4_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.63 s
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_5_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.70 s
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_0_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.71 s
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_6_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.71 s
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_3_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_1_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:43:13 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_7_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.76 s
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.74 s
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:43:13 [backends.py:472] Dynamo bytecode transform time: 10.74 s
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:43:14 [backends.py:462] Using cache directory: /home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_2_0 for vLLM's torch.compile
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:43:14 [backends.py:472] Dynamo bytecode transform time: 10.82 s
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:43:16 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:43:16 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:43:16 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:43:16 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:43:17 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:43:17 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:43:17 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:43:17 [backends.py:161] Cache the graph of shape None for later use
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:43:52 [backends.py:173] Compiling a graph for general shape takes 37.67 s
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:43:52 [backends.py:173] Compiling a graph for general shape takes 37.80 s
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:43:52 [backends.py:173] Compiling a graph for general shape takes 37.86 s
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:43:52 [backends.py:173] Compiling a graph for general shape takes 38.15 s
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:43:53 [backends.py:173] Compiling a graph for general shape takes 38.02 s
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:43:53 [backends.py:173] Compiling a graph for general shape takes 38.20 s
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:43:53 [backends.py:173] Compiling a graph for general shape takes 38.22 s
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:43:53 [backends.py:173] Compiling a graph for general shape takes 38.24 s
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.84 s in total
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.38 s in total
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.50 s in total
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.96 s in total
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 49.00 s in total
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.78 s in total
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.94 s in total
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:44:15 [monitor.py:34] torch.compile takes 48.57 s in total
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.82 GiB
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 136.45 GiB
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.82 GiB
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.82 GiB
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.82 GiB
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.95 GiB
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.82 GiB
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:44:18 [gpu_worker.py:227] Available KV cache memory: 135.82 GiB
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,563,712 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6960.38x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,560,432 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6953.97x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,560,432 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6953.97x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,560,432 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6953.97x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,560,432 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6953.97x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,560,432 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6953.97x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,560,432 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6953.97x
INFO 06-24 23:44:18 [kv_cache_utils.py:715] GPU KV cache size: 3,576,816 tokens
INFO 06-24 23:44:18 [kv_cache_utils.py:719] Maximum concurrency for 512 tokens per request: 6985.97x
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:44:49 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:44:49 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:44:49 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:44:50 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:44:50 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:44:50 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:44:50 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:44:52 [custom_all_reduce.py:196] Registering 10626 cuda graph addresses
[1;36m(VllmWorker rank=7 pid=16509)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=3 pid=16505)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=4 pid=16506)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=1 pid=16503)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=2 pid=16504)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=6 pid=16508)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=5 pid=16507)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=0 pid=16502)[0;0m INFO 06-24 23:44:52 [gpu_model_runner.py:2048] Graph capturing finished in 34 secs, took 1.25 GiB
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527] WorkerProc hit an exception.
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527] Traceback (most recent call last):
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/v1/executor/multiproc_executor.py", line 522, in worker_busy_loop
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     output = func(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]              ^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/v1/worker/gpu_worker.py", line 272, in compile_or_warm_up_model
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     hidden_states=self.model_runner._dummy_run(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return func(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/v1/worker/gpu_model_runner.py", line 1847, in _dummy_run
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     outputs = model(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]               ^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._call_impl(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return forward_call(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/model_executor/models/llama.py", line 581, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     model_output = self.model(input_ids, positions, intermediate_tensors,
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/compilation/decorators.py", line 246, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     model_output = self.forward(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/model_executor/models/llama.py", line 368, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     def forward(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._call_impl(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return forward_call(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_dynamo/eval_frame.py", line 838, in _fn
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return fn(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/fx/graph_module.py", line 830, in call_wrapped
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._wrapped_call(self, *args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/fx/graph_module.py", line 406, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     raise e
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/fx/graph_module.py", line 393, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return super(self.cls, obj).__call__(*args, **kwargs)  # type: ignore[misc]
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._call_impl(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return forward_call(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "<eval_with_key>.162", line 490, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     submod_0 = self.submod_0(l_input_ids_, s0, l_self_modules_embed_tokens_parameters_weight_, l_self_modules_layers_modules_0_modules_input_layernorm_parameters_weight_, l_self_modules_layers_modules_0_modules_self_attn_modules_qkv_proj_parameters_weight_, l_positions_, l_self_modules_layers_modules_0_modules_self_attn_modules_rotary_emb_buffers_cos_sin_cache_);  l_input_ids_ = l_self_modules_embed_tokens_parameters_weight_ = l_self_modules_layers_modules_0_modules_input_layernorm_parameters_weight_ = l_self_modules_layers_modules_0_modules_self_attn_modules_qkv_proj_parameters_weight_ = None
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/compilation/cuda_piecewise_backend.py", line 116, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self.compiled_graph_for_general_shape(*args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_dynamo/eval_frame.py", line 838, in _fn
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return fn(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/aot_autograd.py", line 1201, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return compiled_fn(full_args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/runtime_wrappers.py", line 328, in runtime_wrapper
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     all_outs = call_func_at_runtime_with_args(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/utils.py", line 126, in call_func_at_runtime_with_args
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     out = normalize_as_list(f(args))
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                             ^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/runtime_wrappers.py", line 689, in inner_fn
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     outs = compiled_fn(args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/runtime_wrappers.py", line 495, in wrapper
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return compiled_fn(runtime_args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_inductor/output_code.py", line 460, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self.current_callable(inputs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_inductor/utils.py", line 2404, in run
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return model(new_inputs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_7_0/inductor_cache/g3/cg3lg2wkk5wiziwjc3lcntr4ebepv42vrs4rmlfk5iqnnxfargry.py", line 395, in call
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     triton_poi_fused_cat_2.run(buf5, arg5_1, arg6_1, buf6, triton_poi_fused_cat_2_xnumel, stream=stream7)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_inductor/runtime/triton_heuristics.py", line 956, in run
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return launcher(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "<string>", line 5, in launcher
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/triton/backends/nvidia/driver.py", line 529, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     self.launch(gridX, gridY, gridZ, stream, function, self.launch_cooperative_grid, global_scratch, *args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527] RuntimeError: Triton Error [CUDA]: uncorrectable NVLink error detected during the execution
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527] Traceback (most recent call last):
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/v1/executor/multiproc_executor.py", line 522, in worker_busy_loop
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     output = func(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]              ^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/v1/worker/gpu_worker.py", line 272, in compile_or_warm_up_model
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     hidden_states=self.model_runner._dummy_run(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return func(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/v1/worker/gpu_model_runner.py", line 1847, in _dummy_run
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     outputs = model(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]               ^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._call_impl(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return forward_call(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/model_executor/models/llama.py", line 581, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     model_output = self.model(input_ids, positions, intermediate_tensors,
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/compilation/decorators.py", line 246, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     model_output = self.forward(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/model_executor/models/llama.py", line 368, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     def forward(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._call_impl(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return forward_call(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_dynamo/eval_frame.py", line 838, in _fn
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return fn(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/fx/graph_module.py", line 830, in call_wrapped
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._wrapped_call(self, *args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/fx/graph_module.py", line 406, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     raise e
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/fx/graph_module.py", line 393, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return super(self.cls, obj).__call__(*args, **kwargs)  # type: ignore[misc]
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1751, in _wrapped_call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self._call_impl(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/nn/modules/module.py", line 1762, in _call_impl
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return forward_call(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "<eval_with_key>.162", line 490, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     submod_0 = self.submod_0(l_input_ids_, s0, l_self_modules_embed_tokens_parameters_weight_, l_self_modules_layers_modules_0_modules_input_layernorm_parameters_weight_, l_self_modules_layers_modules_0_modules_self_attn_modules_qkv_proj_parameters_weight_, l_positions_, l_self_modules_layers_modules_0_modules_self_attn_modules_rotary_emb_buffers_cos_sin_cache_);  l_input_ids_ = l_self_modules_embed_tokens_parameters_weight_ = l_self_modules_layers_modules_0_modules_input_layernorm_parameters_weight_ = l_self_modules_layers_modules_0_modules_self_attn_modules_qkv_proj_parameters_weight_ = None
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/vllm/compilation/cuda_piecewise_backend.py", line 116, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self.compiled_graph_for_general_shape(*args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_dynamo/eval_frame.py", line 838, in _fn
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return fn(*args, **kwargs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/aot_autograd.py", line 1201, in forward
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return compiled_fn(full_args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/runtime_wrappers.py", line 328, in runtime_wrapper
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     all_outs = call_func_at_runtime_with_args(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/utils.py", line 126, in call_func_at_runtime_with_args
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     out = normalize_as_list(f(args))
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]                             ^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/runtime_wrappers.py", line 689, in inner_fn
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     outs = compiled_fn(args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_functorch/_aot_autograd/runtime_wrappers.py", line 495, in wrapper
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return compiled_fn(runtime_args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_inductor/output_code.py", line 460, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return self.current_callable(inputs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_inductor/utils.py", line 2404, in run
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return model(new_inputs)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.cache/vllm/torch_compile_cache/b8f58113b1/rank_7_0/inductor_cache/g3/cg3lg2wkk5wiziwjc3lcntr4ebepv42vrs4rmlfk5iqnnxfargry.py", line 395, in call
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     triton_poi_fused_cat_2.run(buf5, arg5_1, arg6_1, buf6, triton_poi_fused_cat_2_xnumel, stream=stream7)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/_inductor/runtime/triton_heuristics.py", line 956, in run
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     return launcher(
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]            ^^^^^^^^^
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "<string>", line 5, in launcher
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]   File "/home/pirillo_google_com/.vllm/lib/python3.12/site-packages/triton/backends/nvidia/driver.py", line 529, in __call__
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527]     self.launch(gridX, gridY, gridZ, stream, function, self.launch_cooperative_grid, global_scratch, *args)
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527] RuntimeError: Triton Error [CUDA]: uncorrectable NVLink error detected during the execution
[1;36m(VllmWorker rank=7 pid=16509)[0;0m ERROR 06-24 23:44:53 [multiproc_executor.py:527] 
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
Failed: Cuda error /workspace/csrc/custom_all_reduce.cuh:594 'uncorrectable NVLink error detected during the execution'
ERROR 06-24 23:44:53 [multiproc_executor.py:140] Worker proc VllmWorker-0 died unexpectedly, shutting down executor.

--- STDERR ---
huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...
To disable this warning, you can either:
	- Avoid using `tokenizers` before the fork if possible
	- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:   0% Completed | 0/30 [00:00<?, ?it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:   3% Completed | 1/30 [00:00<00:05,  4.98it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:   7% Completed | 2/30 [00:00<00:06,  4.39it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  10% Completed | 3/30 [00:00<00:06,  4.12it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  13% Completed | 4/30 [00:00<00:06,  4.03it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  17% Completed | 5/30 [00:01<00:06,  3.99it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  20% Completed | 6/30 [00:01<00:06,  3.97it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  23% Completed | 7/30 [00:01<00:05,  4.43it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  27% Completed | 8/30 [00:01<00:05,  4.29it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  30% Completed | 9/30 [00:02<00:05,  4.19it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  33% Completed | 10/30 [00:02<00:04,  4.05it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  37% Completed | 11/30 [00:02<00:04,  4.21it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  40% Completed | 12/30 [00:02<00:04,  4.35it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  43% Completed | 13/30 [00:03<00:04,  4.23it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  47% Completed | 14/30 [00:03<00:03,  4.17it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  50% Completed | 15/30 [00:03<00:03,  4.12it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  53% Completed | 16/30 [00:03<00:03,  4.28it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  57% Completed | 17/30 [00:04<00:03,  4.23it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  60% Completed | 18/30 [00:04<00:02,  4.22it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  63% Completed | 19/30 [00:04<00:02,  4.17it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  67% Completed | 20/30 [00:04<00:02,  4.14it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  70% Completed | 21/30 [00:05<00:02,  4.10it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  73% Completed | 22/30 [00:05<00:01,  4.26it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  80% Completed | 24/30 [00:05<00:01,  5.12it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  83% Completed | 25/30 [00:05<00:01,  4.80it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  87% Completed | 26/30 [00:06<00:00,  4.57it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  90% Completed | 27/30 [00:06<00:00,  4.39it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  93% Completed | 28/30 [00:06<00:00,  4.31it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards:  97% Completed | 29/30 [00:06<00:00,  4.26it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards: 100% Completed | 30/30 [00:07<00:00,  4.21it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m Loading safetensors checkpoint shards: 100% Completed | 30/30 [00:07<00:00,  4.28it/s]
[1;36m(VllmWorker rank=0 pid=16502)[0;0m 
[rank4]:[W624 23:44:53.207366200 TCPStore.cpp:125] [c10d] recvValue failed on SocketImpl(fd=195, addr=[localhost]:49998, remote=[localhost]:51497): failed to recv, got 0 bytes
Exception raised from recvBytes at /pytorch/torch/csrc/distributed/c10d/Utils.hpp:678 (most recent call first):
frame #0: c10::Error::Error(c10::SourceLocation, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) + 0x98 (0x71039a3785e8 in /home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/lib/libc10.so)
frame #1: <unknown function> + 0x5ba8afe (0x71038a7a8afe in /home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/lib/libtorch_cpu.so)
frame #2: <unknown function> + 0x5baae40 (0x71038a7aae40 in /home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/lib/libtorch_cpu.so)
frame #3: <unknown function> + 0x5bab74a (0x71038a7ab74a in /home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/lib/libtorch_cpu.so)
frame #4: c10d::TCPStore::check(std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&) + 0x2a9 (0x71038a7a51a9 in /home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/lib/libtorch_cpu.so)
frame #5: c10d::ProcessGroupNCCL::heartbeatMonitor() + 0x379 (0x7103362f3989 in /home/pirillo_google_com/.vllm/lib/python3.12/site-packages/torch/lib/libtorch_cuda.so)
frame #6: <unknown function> + 0xecdb4 (0x7103262ecdb4 in /lib/x86_64-linux-gnu/libstdc++.so.6)
frame #7: <unknown function> + 0x9caa4 (0x7103a789caa4 in /lib/x86_64-linux-gnu/libc.so.6)
frame #8: <unknown function> + 0x129c3c (0x7103a7929c3c in /lib/x86_64-linux-gnu/libc.so.6)

[rank4]:[W624 23:44:53.217996061 ProcessGroupNCCL.cpp:1659] [PG ID 0 PG GUID 0 Rank 4] Failed to check the "should dump" flag on TCPStore, (maybe TCPStore server has shut down too early), with error: failed to recv, got 0 bytes
